server/src/controllers/Events.js

```diff
@@ -66,22 +66,24 @@ module.exports = {
         };
       }
       const event = await Event.findByIdAndUpdate(eventId, req.body);
-      res.json({
+      return res.status(200).json({
         id: event._id
       });
     } catch (err) {
-      res.status(400).send({
-        err: err
-      });
+      return res.status(400).json({
+        message: err.message,
+      })
```

Comment:
Since 400 is a bad request, it'd be great if you could replace the key with "err" or "error" to signify that this is, in fact, an error

Reply:
Alright.